kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Tab.java

```diff
@@ -22,6 +22,8 @@
 
     public void refresh();
 
+    public void clear();
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"